diff -ruN -x __pycache__ -x obj -x '*~' c-blosc-1.21.1.orig/cmake/FindLZ4.cmake c-blosc-1.21.1/cmake/FindLZ4.cmake
--- c-blosc-1.21.1.orig/cmake/FindLZ4.cmake	2021-10-06 10:16:29.000000000 +0200
+++ c-blosc-1.21.1/cmake/FindLZ4.cmake	2022-05-09 21:19:55.255608014 +0200
@@ -1,6 +1,6 @@
 find_path(LZ4_INCLUDE_DIR lz4.h)
 
-find_library(LZ4_LIBRARY NAMES lz4)
+find_library(LZ4_LIBRARY NAMES lz4 liblz4)
 
 if (LZ4_INCLUDE_DIR AND LZ4_LIBRARY)
     set(LZ4_FOUND TRUE)
